### ISSUE TITLE
The sample code for querying tables looks incorrect

### ISSUE THREAD

**Issue by ndchorley** [orangy/squash#6]
In the readme, there is

```kotlin
object Citizens : TableDefinition() {
    val id = varchar("id", 10).primaryKey()
    val name = varchar("name", length = 50)
    val cityId = reference(Cities.id, "city_id").nullable()
}
...

val row = query(Citizens)
...
```

This won't compile - `query` takes an argument of type `QueryObject` and `TableDefinition` doesn't inherit from that.

**Comment by jstr14** [comment]
Same error here, @orangy Can you explain us something about it?

**Comment by 0ff1ane** [comment]
Looks like you'd have to use "from"
`from(Citizens).select(Citizens.name, Citizens.id).where { Citizens.name eq "name" }
`

I'm looking at the benchmarks file: [https://github.com/orangy/squash/blob/master/squash-benchmarks/src/org/jetbrains/squash/benchmarks/QueryBenchmark.kt](url)

**Comment by orangy** [comment]
May be you need to import another package – there are few `query` functions there, and that's something we should fix.

**Comment by brezinajn** [comment]
I'm not able to find correct query import. Would you please provide the correct import?

**Comment by orangy** [comment]
Yes, @rascala was right, and I was wrong. You need `from` there.

**Comment by codesplode** [comment]
Just going through issues and it looks like this is already done. @orangy , anything else you'd like done in order to close this issue? I can certainly provide more examples and documentation now that I've found my way around if you like.
